src/main/java/frc/robot/CommandSequences.java

```diff
@@ -40,120 +40,191 @@ public class CommandSequences {
+        reefNodes[5] = simplePose(5.294, 2.962, 120); //Reef Position F
+        reefNodes[6] = simplePose(5.81, 3.86, 180); //Reef Position G
+        reefNodes[7] = simplePose(5.81, 4.192, 180); //Reef Position H
+        reefNodes[8] = simplePose(5.288, 5.084, 240); //Reef Position I
+        reefNodes[9] = simplePose(5.006, 5.253, 240); //Reef Position J // was 5, 
+        reefNodes[10] = simplePose(3.972, 5.249, 300); //Reef Position K
+        reefNodes[11] = simplePose(3.686, 5.085, 300); //Reef Position L
+
+
+        leftHumanPlayer = simplePose(1.127, 6.982, 306);
+        rightHumanPlayer = simplePose(1.127, 1.035, 54);
+
+        middle =  simplePose(7.115, 4, 180);
+        rightReefPassage = simplePose(5.8, 1.7, 75);
+        leftReefPassage = simplePose(5.8, 7, 285);
+
+        processor = simplePose(6.350, 0.550,270);
     }
-    
-    public Command driveAndPlaceOneFromMiddle(SwerveSubsystem swerveSubsystem, ElevatorSubsystem elevatorSubsystem, CoralDispenserSubsystem coralDispenserSubsystem){
-        swerveSubsystem.setOdometry(middle.toFieldPose2d());
+
+    //test auto to show how coral collection works not to be used
+    public Command coralcollectionautotest(CoralDispenserSubsystem coralDispenserSubsystem){
         return new SequentialCommandGroup(
-            new SwerveDriveToPointCmd(swerveSubsystem, reefNode('G')),
-            new AutoElevatorCommand(elevatorSubsystem, ElevatorConstants.kElevatorL4Height),
-            new AutoCoralDispenseCommand(coralDispenserSubsystem)
+            new ParallelCommandGroup(
+                new CollectCoralCmd(coralDispenserSubsystem),
+                coralDispenserSubsystem.seecoral == true ?
+                    new SwerveDriveToPointCmd(null, reefNode('C')) : null
+            )
         );
     }
-    public Command driveAndPlaceOneFromLeft(SwerveSubsystem swerveSubsystem, ElevatorSubsystem elevatorSubsystem, CoralDispenserSubsystem coralDispenserSubsystem){
+    //Starting Position to reef
+
+    public Command CageOneToH(SwerveSubsystem swerveSubsystem){
         swerveSubsystem.setOdometry(cageNodes[0].toFieldPose2d());
-        return new SequentialCommandGroup(
-            new SwerveDriveToPointCmd(swerveSubsystem, reefNode('J')),
-            new AutoElevatorCommand(elevatorSubsystem, ElevatorConstants.kElevatorL4Height),
-            new AutoCoralDispenseCommand(coralDispenserSubsystem),
-            new AutoElevatorCommand(elevatorSubsystem, 0)
-        );
+        return new SwerveDriveToPointCmd(swerveSubsystem, reefNode('H'));
+    }
+
+    public Command CageOneToI(SwerveSubsystem swerveSubsystem){
+        swerveSubsystem.setOdometry(cageNodes[0].toFieldPose2d());
+        return new SwerveDriveToPointCmd(swerveSubsystem, reefNode('I'));
+    }
+
+    public Command CageTwoToH(SwerveSubsystem swerveSubsystem){
+        swerveSubsystem.setOdometry(cageNodes[1].toFieldPose2d());
+        return new SwerveDriveToPointCmd(swerveSubsystem, reefNode('H'));
+    }
```

Comment:
```suggestion
    public Command setOdometryAndSwerveToPoint(SwerveSubsystem swerveSubsystem, int cageNode, char reefNodeChar){
        swerveSubsystem.setOdometry(cageNodes[cageNode].toFieldPose2d());
        return new SwerveDriveToPointCmd(swerveSubsystem, reefNode(reefNodeChar));
    }
```

Reply:
This could then be called for all the getAutonomousCommand calls

Reply:
I suppose given our time crunch this is something we can refactor later but just try to look for that repetition happening in your code and see where maybe you need to encapsulate that and build a loop/function/class etc